### ISSUE TITLE
Textbox readonly color contrast issue

### ISSUE THREAD

**Issue by edvardsniedre** [eBay/skin#1775]
Color contrast issue on `readonly textbox`. Identified using axe DevTools. Tested on Skin v13.7.1

<img width="1214" alt="Screenshot 2022-06-15 at 15 10 13" src="https://user-images.githubusercontent.com/7050068/173852681-c8cae0d1-ca1a-49bf-9cc2-fcdf15b27d6d.png">

**Comment by PrashantAshok** [comment]
The text is not visible in Skin 14. Reproducible in skin docs site https://opensource.ebay.com/skin/#textbox

<img width="666" alt="Screen Shot 2022-07-11 at 10 23 18 AM" src="https://user-images.githubusercontent.com/5200733/178321985-e6cbf863-bb44-482a-82f4-4a05f1e08853.png">

**Comment by PrashantAshok** [comment]
Seeing another issue, the color of textbox is lighter when `focus`ed compared to non-`focus`ed state. 

<img width="360" alt="Screen Shot 2022-07-11 at 12 19 16 PM" src="https://user-images.githubusercontent.com/5200733/178347820-5f242c96-f636-41b2-9b89-d3bc646deb98.png">
<img width="279" alt="Screen Shot 2022-07-11 at 12 19 21 PM" src="https://user-images.githubusercontent.com/5200733/178347824-15d47788-13d4-400c-86bc-cd767c6d874d.png">
